mm2src/coins/eth/eth_tests.rs

```diff
@@ -249,16 +249,16 @@ fn send_and_refund_erc20_payment() {
     }));
 
     let payment = coin
-        .send_maker_payment(
-            0,
-            (now_ms() / 1000) as u32 - 200,
-            &DEX_FEE_ADDR_RAW_PUBKEY,
-            &[1; 20],
-            "0.001".parse().unwrap(),
-            &coin.swap_contract_address(),
-            &[],
-            &None,
-        )
+        .send_maker_payment(SendMakerPaymentArgs {
```

Comment:
If you create an instance of `SendMakerPaymentArgs` before the `send_maker_payment` method call, it will take 2 lines less:
```rust
let args = SendMakerPaymentArgs {
            time_lock_duration: 0,
            time_lock: (now_ms() / 1000) as u32 - 200,
            pubkey: &DEX_FEE_ADDR_RAW_PUBKEY,
            secret_hash: &[1; 20],
            amount: "0.001".parse().unwrap(),
            swap_contract_address: &coin.swap_contract_address(),
            swap_unique_data: &[],
            payment_instructions: &None,
        };
coin.send_maker_payment(args).wait().unwrap();
```
It's not so much for one function call, but this approach can save around 10 lines in this file

Reply:
done in tests